### ISSUE TITLE
[BUG] chunks missing from planet surfaces 

### ISSUE THREAD

**Issue by TIKIRobo** [tsunamayo/Starship-EVO#5283]
### Short Description of the issue:
When driving or flying around near planets chunks of the surface will just disappear, the further away I am from the surface generally the larger the chunks and usually the faster I move the more often they appear, this happens every time i visit a planet 
in propeach's report #5273 I mistakenly reported this issue as the same one he was having 
...

### Steps to reproduce the issue:
fly or drive near the planets surface quickly 
...

### Environment
- Starship EVO version: 23w35b
- Graphics Card: AMD Radeon RX 5600 XT

### Additional information:

https://github.com/tsunamayo/Starship-EVO/assets/56371294/8c554788-0ea1-42d9-83b0-fd53c5a0061e

look at the ground in this video
[Player.log](https://github.com/tsunamayo/Starship-EVO/files/12569899/Player.log)

these next two video were recorded right after a restart and with only a few seconds in between each clip 

https://github.com/tsunamayo/Starship-EVO/assets/56371294/c09cff37-25e7-439c-b2c0-c2a3ea1724e8

https://github.com/tsunamayo/Starship-EVO/assets/56371294/a7278dee-d4da-4b8a-83ce-93c1f5d70584

output from that session 

[Player.log](https://github.com/tsunamayo/Starship-EVO/files/12569924/Player.log)

**Comment by ProPeach** [comment]
I also have these missing chunks on planet surfaces. They show up on the asteroids in gas giants rings too which I mentioned briefly in the report Tiki points out, but it's great to have a dedicated post for this particular issue as it is a bit different to the rapid flickering squares in #5273.

https://github.com/tsunamayo/Starship-EVO/assets/27857711/a36cbe7a-33ed-442d-b5f1-16702caf3014


I have an RX 6700XT, so Tiki and I both run on AMD. I know you mentioned in the other post that you didn't see the flickering squares Tsuna, think this might be a brand specific issue? iirc you're still running an Nvidea 3000 series

**Comment by tsunamayo** [comment]
@ProPeach @TIKIRobo Thanks, if you guys could ask other (if you are on Discord) on whether they get this, and if it what is their GFX card. I dont get that at all unfortunately...
Cheers

**Comment by TIKIRobo** [comment]
Asking now! hopefully some people will get back to me soon

**Comment by Xerma** [comment]
My gpu is an RTX 3070 and I haven't seen any missing chunks and just flew all the way around a planet

**Comment by tsunamayo** [comment]
@asanagi-sae Yes I need to fix this it is when the planets switch from Far to Close camera (I have a 2 camera setup)

**Comment by Tisdag1** [comment]
I also get this happening. My card is an AMD Radeon RX 6700M

**Comment by TIKIRobo** [comment]
https://github.com/tsunamayo/Starship-EVO/assets/56371294/fdf1d428-e8a3-451f-8f03-d47c4a8d8819

not sure if this is related but if you watch closely in this video you can see chunks of the terrain that are being culled because they're behind my ship but then loading in slowly once they're in view 
hopefully this helps @tsunamayo 

also something worth mentioning is that players like me and peach see massive frame drops when going from space to planets, i go from a stable 70 ish to an unstable 45, peach also has his fps cut in half

**Comment by tsunamayo** [comment]
@asanagi-sae @TIKIRobo Hi, thanks for letting me know this is something I also noticed. It is a different issue though, as it is related to culling. They just skip a single frame, AMD is a more serious issue...
